client/src/utils/secureAuth.js

```
@@ -0,0 +1,84 @@
+			);
+			return JSON.parse(jsonPayload);
+		} catch (error) {
+			return null;
+		}
+	}
+
+	// Check if user is authenticated
+	static isAuthenticated() {
+		return this.getToken() !== null && this.getUserData() !== null;
+	}
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)